libs/barista-components/radio/src/radio.ts

```diff
@@ -54,7 +54,12 @@ export interface DtRadioChange<T> {
 // Boilerplate for applying mixins to DtRadioButton.
 export class DtRadioButtonBase {
   /** Whether the radio button is disabled */
-  disabled: boolean;
+  // These must be properties instead of fields to avoid a TS error
+  // see https://github.com/microsoft/TypeScript/pull/37894
+  get disabled(): boolean {
+    return false;
+  }
+  set disabled(_: boolean) {}
```

Comment:
I think we should implement this correctly here as well. Might be confusing to the next one that we always return false here. I know that we extend from it and override the disabled getter and setter but still this is odd and might lead to bugs if someone does not override them.